configure.ps1

```diff
@@ -63,6 +67,10 @@ Invoke-BuildStep 'Cleaning package cache' {
     Clear-PackageCache
 } -skip:(-not $CleanCache) -ev +BuildErrors
 
+Invoke-BuildStep 'Disabling strong name verification' {
```

Comment:
I think this will make `configure.ps1` launch another elevated process every time right?  Or does our script to disable strong name validation only run as elevated the one time it does its thing?

Reply:
It only launches another process if:

1. Validation for the two keys are not already disabled
2. the current process isn't already elevated

So, you should only see it once per machine, as long as you don't re-enable validation (for example, running `sn.exe -Vx`)

https://github.com/NuGet/NuGet.Client/blob/81321332739c442de3c7e640720bb7abf31f42ca/scripts/utils/DisableStrongNameVerification.ps1#L5-L12

Reply:
I was surprised that it doesn't need to run elevated to read the HKLM hive of the registry but apparently its fine.

Are we concerned that members of the development community will have this setting on their machines if they clone our repo and run `configure.ps1`?  Strong name validation is disabled for us when we install internal previews so I guess I'm just concerned that we'll be running this when it might not be needed?